transports/webrtc-websys/Cargo.toml

```
@@ -0,0 +1,69 @@
+[package]
+name = "libp2p-webrtc-websys"
+version = "0.1.0"
+authors = ["Doug Anderson <DougAnderson444@peerpiper.io>"]
+description = "Web-Sys WebRTC transport for libp2p"
```

```suggestion
description = "WebRTC for libp2p under WASM environment"
```

This would be consistent with what we did for `libp2p-webtransport-websys`.

---

Done

---

Resolved